### PR TITLE
Updates to wallet and full node for Testnet wave2

### PR DESCRIPTION
**Pull request by randall-Mysten**

### CODE REVIEW

File: doc/src/build/fullnode.md [hunk 1]
```diff
@@ -185,7 +191,14 @@ Source](#building-from-source), update your Full node as follows:
     ```shell
     git checkout -B devnet --track upstream/devnet
     ```
-1. Download the latest [`genesis`](https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob) state for Devnet as described above.
+1. Download the latest Genesis blob:
```
Comment (ronny-mysten) [inline doc/src/build/fullnode.md:194]: genesis elsewhere.

File: doc/src/explore/wallet-browser.md [hunk 2]
```diff
@@ -64,18 +65,15 @@ Sui Wallet prompts you to enter your password when you open it after the first u
...
+When you first open the wallet, you have no coins in it. You can add test SUI coins to your wallet using the faucet in Discord.
 
 **To get SUI test coins through Discord**
 1. Click **Coins**.
 1. Click the small clipboard icon next to your address to copy it.
    It's near the top of the wallet and starts with 0x.
-1. Open the Sui [devnet-faucet](https://discord.com/channels/916379725201563759/971488439931392130) channel in Discord. You must have an active Discord account that has joined the Sui Discord to access the channel.
+1. Go to the Discord faucet channel for the network you use:
+   * [devnet-faucet](https://discord.com/channels/916379725201563759/971488439931392130) channel in Discord.
+   * [testnet-faucet](https://discord.com/channels/916379725201563759/1037811694564560966)
```
Comment (ronny-mysten) [inline doc/src/explore/wallet-browser.md:76]:
```suggestion
   * [testnet-faucet](https://discord.com/channels/916379725201563759/1037811694564560966).
```
or, ...channel in Discord.

File: doc/src/explore/wallet-browser.md [hunk 3]
```diff
@@ -132,6 +130,23 @@ You can send coins from your wallet to another address.
...
+1. Select a validator to stake with.
+1. Choose an amount of SUI to stake.
+   Be sure to enter an amount that leaves enough SUI in your wallet to cover gas fees.
+1. Click **Stake Now**.
+
+Your stake starts earning rewards at the start of the next epoch.
+
+## View current stake
+
+To view details about your current stakes, click Currently Staked on the Coins tab of the Wallet. Details include: the amount you staked, the validator you chose, amount earned, and the validator commission.
```
Comment (ronny-mysten) [inline doc/src/explore/wallet-browser.md]:
```suggestion
To view details about your current stakes, click **Currently Staked** on the **Coins** tab of the Wallet. Details include: the amount you staked, the validator you chose, amount earned, and the validator commission.
```

File: doc/src/build/fullnode.md (hunk 1 above)
Comment (randall-Mysten) [reply doc/src/build/fullnode.md]: ty

File: doc/src/explore/wallet-browser.md (hunk 3 above)
Comment (randall-Mysten) [reply doc/src/explore/wallet-browser.md]: sorry I added that after I created the PR, thanks for catching

**Review by ronny-mysten — APPROVED** [review]

**Pull request merged**